berkshelf.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.name                      = "berkshelf"
   s.require_paths             = ["lib"]
   s.version                   = Berkshelf::VERSION
-  s.required_ruby_version     = ">= 2.7.0"
+  s.required_ruby_version     = ">= 3.3"
```

Comment:
This will break everything that depends on it. `">= 3.0"` is the best we can do.